test/plugins/test_plugins_handlers.py

```diff
@@ -100,3 +100,21 @@ def test_get_label_entrypoint(plugin_tmpfile):
     assert meta['label'] == 'plugin label'
     assert meta['type'] == handlers.EntryPointPlugin.PLUGIN_TYPE
     assert meta['source'] == 'test_plugin = file_mod'
+
+
+def test_entrypoint_plugin_get_version_does_not_raise(plugin_tmpfile):
```

Comment:
To be on brand with the rest of the file, I suggest this nitpicky modification:

```suggestion
def test_get_version_entrypoint_package_does_not_match(plugin_tmpfile):
```